platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/SwirldStateManager.java

```diff
@@ -126,33 +127,38 @@ public SwirldStateManager(
         initialState(state);
     }
 
+    public void prehandleApplicationTransactions(final GossipEvent event) {
```

Comment:
Missing docs for public method

Reply:
Added.

```
    /**
     * Prehandles application transactions. Similar to {@link #prehandleApplicationTransactions(EventImpl)} but accepts
     * a {@link GossipEvent} instead of an {@link EventImpl}.
     *
     * @param event the event to handle
     */
```